src/main/java/bio/terra/common/PdaoLoadStatistics.java

```diff
@@ -9,7 +10,11 @@ public class PdaoLoadStatistics {
   private final Instant startTime;
   private final Instant endTime;
 
-  public PdaoLoadStatistics(long badRecords, long rowCount, Instant startTime, Instant endTime) {
+  public PdaoLoadStatistics(
```

Comment:
Either the spring boot or jackson update caused the constructor to no longer be found during deserialization. The tests were failing trying to deserialize this class from the flightmap. Adding constructor annotations fixes the problem. I also tried converting this to a `record`, which works, but it serializes differently and we'd no longer be able to deserialize data from older flights.